core/common/src/main/java/alluxio/util/network/NetworkAddressUtils.java

```diff
@@ -83,7 +83,7 @@ private NetworkAddressUtils() {}
      * underlying Raft implementation doesn't differentiate between bind and connect hosts.
      */
     MASTER_RAFT("Alluxio Master Raft service", PropertyKey.MASTER_HOSTNAME,
-        PropertyKey.MASTER_HOSTNAME, PropertyKey.MASTER_EMBEDDED_JOURNAL_PORT),
+        PropertyKey.MASTER_BIND_HOST, PropertyKey.MASTER_EMBEDDED_JOURNAL_PORT),
```

Comment:
can you update the comment? it says `The bind and connect hosts are the same ` which I think is inaccurate now. This means the new comment should explain why they are different.

Reply:
@gpang updated the comment

Reply:
I have a question, if this is true, then why don't other hosts have to change for other services in this file?

Reply:
@gpang seems like right now master and job-master both start the raft journal system and bind to `PropertyKey.MASTER_HOSTNAME`. `PropertyKey.JOB_MASTER_HOSTNAME` is ignored when starting `CopycatGrpcServer` in the job master process. @ggezer sounds like a bug? in my k8s environment it worked because  the job master binds to the master bind address as well

Reply:
reverting the original change, I just verified that I can bind to the service hostname w/ a k8s spec change.
regardless job_master_hostname vs master_hostname might be an independent issue which we might need to address in a separate PR

Reply:
hmm, might have tested wrong. do not review yet, a separate bind might still be needed